### PR TITLE
[full-ci] Migrate deny-acl UI code from CERNbox

### PR DESCRIPTION
**Pull request by pascalwengerter**
Porting over https://github.com/cernbox/web/commit/1d75877919e862fdecfdff3bae366ede3951ba1a

## Description
See changelog item

## Types of changes
- [X] New feature (non-breaking change which adds functionality)

## Open tasks:
- [ ] Merged https://github.com/cs3org/reva/pull/3011
- [ ] Corresponding Reva-bump in oCIS
- [ ] Adding capability-based rendering of "deny" option incl. polishing of code changes
- [ ] Adding unit & E2E tests

### CODE REVIEW

File: packages/web-app-files/src/helpers/resources.ts [hunk 1]
```diff
@@ -238,6 +242,9 @@ export function buildSpace(space) {
     isReceivedShare: function () {
       return false
     },
+    canDeny: function () {
+      return false // FIXME
```
Comment (pascalwengerter) [inline packages/web-app-files/src/helpers/resources.ts:246]: Question what is "to be fixed" here

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/28326/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/28326/tracing/re-sharing-alice-2022-9-14-10-31-11.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/28326/tracing/re-sharing-brian-2022-9-14-10-31-21.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

**Comment by diocas** [comment]
We'll check a bit later and provide feedback

**Review by diocas — APPROVED** [review]
All seems to be working fine on our side. thanks

**Pull request merged**